apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -269,4 +270,22 @@ public static Character parseCharacter(String charText) {
       return Optional.of(parseCharacter(charText));
     }
   }
+
+  /**
+   * @param inputString {@link String} that possible contains invalid characters
+   * @return an {@link String} with a null representation stripped from it if the input has a null
+   *     field representation.
+   */
+  public static String removeInvalidCharacters(String inputString) {
+    String verifiedInputString = inputString;
+
+    // Null Field
+    String nullFields = "\\0";
```

Comment:
I've been staring at this for a while now and I _think_ it's incorrect? What you're searching for and replacing here is a two-char sequence: a ``\'` followed by a `'0'`. Is that what is needed? Are we getting backslash-zero character sequences from the CCW that we need to strip out?

Or are we really getting null bytes hiding in strings? If we _are_, then you'd want to change this to `"\0"` or perhaps `"\0x00`", instead. You'd also likely need to update the sample RIF. I'd _guess_ that you'd need to use a hex editor to add the null bytes, as I don't think most text editors would let you type them.